models/standalone-price/src/standalone-price-draft/presets/withTiers.ts

```diff
@@ -0,0 +1,21 @@
+import { MoneyDraft, PriceTierDraft } from '@commercetools-test-data/commons';
+import type { TStandalonePriceDraftBuilder } from '../../types';
+import StandalonePriceDraft from '../builder';
+
+const withTiers = (currencyCode: string): TStandalonePriceDraftBuilder => {
```

Comment:
Should the staged value have the same currency too? Maybe @ARRIOLALEO and @dogayuksel can throw light on this question.

Reply:
It is possible to have a different currency for the staged price, it doesn't contradict the specs.
I like the approach of not having to explicitly name it as a unified currency thing and instead having the API spec as a guideline. If it's `withTiers` then the tiers should be valid.

I see the benefit of having a homogeneous currency but I think of it as an additional preset when we'll need for one.

Reply:
AFAIK we will not allow the customer to change the currency and the most logic will be that a price contains only one currency, if they need staged price to have another currency means it is in another country, the logic will be that the customer create a new price, in my opinion we have too keep the logic one price one type of currency.